cubeviz/controls/slice.py

```diff
@@ -31,6 +35,11 @@ def __init__(self, cubeviz_layout):
         self._slice_slider.setEnabled(False)
         self._slice_textbox.setEnabled(False)
         self._wavelength_textbox.setEnabled(False)
+        self._wavelength_textbox_label.setWordWrap(True)
+
+        # This should be used to distinguised between observed and rest wavelengths
+        # We are not going to enforce what the name should be at this level.
+        self._wavelength_label_text = 'Obs Wavelength'
```

Comment:
Since this string and `Rest Wavelength` are each used in a few different places, it might be worthwhile to define them in global constants somewhere.

Reply:
Added globals.